meson.build

```diff
@@ -0,0 +1,9 @@
+project('sphgeom', 'cpp')
+
+subdir('src')
+
+pkg_mod = import('pkgconfig')
```

Comment:
In addition to creating a pkg-config file, you may also want to create a `declare_dependency()` that adds the incdir and the sphgeom library, as an interface, which then allows using this meson.build as a Meson subproject with automatic dependency fallback.